v23.1/qlik.md

```diff
@@ -84,6 +84,8 @@ To use a {{ site.data.products.serverless }} cluster as the target endpoint, set
 - To perform both an initial load and continuous replication of ongoing changes to the target tables, select **Full Load** and **Apply Changes**. This minimizes downtime for your migration.
 - To perform a one-time migration to CockroachDB, select **Full Load** only.
 
+Select **TRUNCATE before loading** or **Do nothing** to preserve the schema you manually created.
```

Comment:
Nit: I would maybe keep the order of the action and reason you had in the previous sentences so that I know what the aim of the action is first, like:

> To preserve the schema you manually created, select **TRUNCATE before loading** or **Do nothing**.

I could also understand the argument that it's a lot of sentence with the same flow and it might be a bit repetitive!